Dockerfile

```diff
@@ -24,18 +24,13 @@ RUN apt-get update && \
     curl -L https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64 -o /usr/local/bin/goss && \
     chmod +x /usr/local/bin/goss && \
     apt-get remove --purge -yq \
-        curl \
-    && \
-    apt-get autoclean -y && \
-    apt-get autoremove -y && \
-    rm -rf /var/lib/{cache,log}/ && \
-    rm -rf /var/lib/apt/lists/*.lz4 && \
-    rm -rf /tmp/* /var/tmp/*
+      curl
 
 # Overlay the root filesystem from this repo
 COPY ./container/root /
 
-RUN goss -g goss.base.yaml validate
+RUN goss -g goss.base.yaml validate && \
+    /bin/bash /clean.sh
```

Comment:
its very important that this clean runs as *part* of the install. Adding and removing files in different Docker layers doesn't actually save space. You might want to copy the individual `clean.sh` file into the container ahead of doing all the installation